pandas/core/groupby/grouper.py

```diff
@@ -893,7 +897,7 @@ def get_grouper(
 
     # a passed-in Grouper, directly convert
     if isinstance(key, Grouper):
-        grouper, obj = key._get_grouper(obj, validate=False)
+        grouper, obj = key._get_grouper(obj, validate=False, observed=observed)
```

Comment:
this looks reasonable but unrelated to the rest of the PR?

Reply:
Thanks - indeed, this can be removed.